crates/kitsune_p2p/kitsune_p2p/src/gossip/simple_bloom.rs

```diff
@@ -142,7 +142,7 @@ struct NodeInfo {
 }
 
 pub(crate) enum HowToConnect {
-    Con(Tx2ConHnd<wire::Wire>),
+    Con(Tx2ConHnd<wire::Wire>, TxUrl),
```

Comment:
@freesig how is the new `remote_url` here different from the url in the `Tx2ConHnd`? Can you add a comment about what the two urls are for?

Reply:
It's to get around the issue that once the connection is closed we can't access the url anymore so we need to track it here. I can add a comment.